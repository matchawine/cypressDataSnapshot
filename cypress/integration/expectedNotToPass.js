describe("Jest snapshot test", function () {
  it("Test is not passing", () => {
    cy.toMatchSnapshot({ test: false })
  })
})
