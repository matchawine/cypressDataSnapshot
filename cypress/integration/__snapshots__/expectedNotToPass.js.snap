// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest snapshot test Test is not passing 1`] = `
Object {
  "test": true,
}
`;